tests/test_interface.py

```diff
@@ -36,6 +60,12 @@ def runner(capfd):
     return Runner(capfd)
```

Comment:
Do you still need to pass in pytest's `capfd`, or can you use your IO wrapper somehow?